bin/check-es-query-ratio.rb

```diff
@@ -240,7 +240,7 @@ def kibana_info
       "#{URI.escape(Time.at(start_time).utc.strftime kibana_date_format)}',mode:absolute,to:'" \
       "#{URI.escape(Time.at(end_time).utc.strftime kibana_date_format)}'))&_a=(columns:!(_source),index:" \
       "#{URI.escape(index)},interval:auto,query:(query_string:(analyze_wildcard:!t,query:'" \
-      "#{URI.escape(config[:query])}')),sort:!('@timestamp',desc))&dummy"
+      "#{URI.escape(config[:query])}')),sort:!('#{config[:timestamp_field]}',desc))"
```

Comment:
replacing with the option makes sense, why remove the `&dummy` this makes it a breaking change most likely. Could we change that here? If not I am gonna need to test it on es clusters 2.x and 5.x to verify if it's a breaking change. If you can give me some documentation showing that its an ES 1.x thing if so I am good I just need to understand the nature of the breaking change.